src/Sarif.Multitool.Library/MergeCommand.cs

```diff
@@ -179,7 +183,14 @@ private async Task<bool> MergeSarifLogsAsync()
                                     };
                                 }
 
-                                key = CreateRuleKey(result.RuleId, run);
+                                if (_options.MergeRuns)
```

Comment:
>MergeRuns [](http://example.com/codeflow?start=45&length=9)

Acccording to `merge-runs` argument description
```
 [Option( 
     "merge-runs", 
     HelpText = "Merge runs of the same tool + verion combination (requires " + 
                "eliding run-specific details such as invocations data.")] 
 public bool MergeRuns { get; set; } 
```
If `merge-runs` is false (default), the merged log runs are aggregated by key `ruleId+tool name+tool version`.
if it is set to true, the merged log runs will be aggregated by key `tool name+tool version`. This means all results produced by same tool and same version will be merged into 1 single run.

Reply:
Yes, your understanding is correct. Do we really have the term `verion` burned into our help? If so, can you please correct this to `version`?

Reply:
fixed typo.